client/keys/add.go

```diff
@@ -62,13 +66,13 @@ the flag --nosort is set.
 	cmd.Flags().Bool(flagNoSort, false, "Keys passed to --multisig are taken in the order they're supplied")
 	cmd.Flags().String(FlagPublicKey, "", "Parse a public key in bech32 format and save it to disk")
 	cmd.Flags().BoolP(flagInteractive, "i", false, "Interactively prompt user for BIP39 passphrase and mnemonic")
-	cmd.Flags().Bool(client.FlagUseLedger, false, "Store a local reference to a private key on a Ledger device")
+	cmd.Flags().Bool(flags.FlagUseLedger, false, "Store a local reference to a private key on a Ledger device")
```

Comment:
So I love the idea of the `flags` package, but what about making the flags in that package `structs`:

```
type Flag interface {
    Type() string
    Description() string
    LongFlag() string
    ShortFlag()
    Register(cmd cobra.Command)
}
```

Reply:
oh yeah we should probably rename the packages to not conflict with the flags namespace... maybe?

Reply:
Yes, that's exactly what we should be doing - I'd not block this PR though

Reply:
@jackzampolin I think I misunderstood you when I made my first comment. In regards to using `pflags` _we already do_ unless I'm missing something. There are only 2 places in the entire repo were we import regular flags (`simapp/sim_test.go` and `contrib/runsim/main.go`)

Reply:
jack [2:44 PM]
Oh I was saying that we could refactor all of the `client.Flag*` defs (currently `string` type) to be `*pflag.Flag` type

which would be a separate PR
Then when registering them we could pass an `[]*pflag.Flag{}` to the `cmd`
And the `client/flags` package could potentially have some standard groupings
Anyway, not a big deal
It would be a separate PR.
Any thoughts on that?

fp4k [2:50 PM]
oh I see, yeah well we wouldn’t want to remove the string (because we still need it to access the flag value using viper) but we could include an additional variable which is the actual `pflag` definition. I think that makes sense as an additional PR as you’ve mentioned